operator/bundle_helpers/README.md

```diff
@@ -2,20 +2,18 @@
 
 For hermetic builds with Konflux, we need to provide the full list of resolved dependencies in `requirements.txt`.
 The dependency source files will be prefetched with Cachi2 and made available to the container image build.
-Because GHA/upstream is running on a different Python version, it uses `requirements-gha.txt` to mange its own dependencies.
-Follow the procedure below after any dependencies change for successful builds in Konflux and mirror those changes into the `requirements-gha.txt`.
+Follow the procedure below after any dependencies change for successful builds in Konflux.
 
 ## Prepare the fully resolved requirements files for Cachi2
 
 ### Prerequisite
 
-Run the steps inside a container of the same image as the operator-bundle builder stage.
+Run the steps inside a container of the same image as the [operator-bundle builder stage](./konflux.bundle.Dockerfile).
```

Comment:
The link doesn't work (you can try [here](https://github.com/stackrox/stackrox/blob/tm/operator-bundle-helpers-improvements/operator/bundle_helpers/README.md#prerequisite)) because the file is one level up.

Reply:
Done in https://github.com/stackrox/stackrox/pull/14257/commits/817a312c46b790f483f0e147f223493ec2872f67